atomicapp/providers/openshift.py

```diff
@@ -449,6 +460,14 @@ def _get_config_values(self):
                            answers[k])
                     logger.error(msg)
                     raise ProviderFailedException(msg)
-        return (result["providerapi"],
-                result["access_token"],
-                result["namespace"])
+
+        logger.debug("config values: %s" % result)
+
+        # set config values
+        self.providerapi = result[PROVIDER_API_KEY]
+        self.access_token = result[ACCESS_TOKEN_KEY]
+        self.namespace = result[NAMESPACE_KEY]
+        if result[PROVIDER_CA_KEY]:
+            self.ssl_verify = result[PROVIDER_CA_KEY]
+        else:
+            self.ssl_verify = result[PROVIDER_TLS_VERIFY_KEY]
```

Comment:
yep you are right, websockets library we use uses standard python ssl lib this has separate parameters for disabling verification and passing CA certificate.

I'm going to break them up as you suggested